snap/hooks/configure

```diff
@@ -62,6 +62,12 @@ vt=0
 EOT
 fi
 
+
+if [ -e "$SNAP/graphics/lib/mir-quirks" ]
```

Comment:
I'd probably put this in `graphics/etc/mir-quirks`? I don't know if there's much value in that other than personal “configuration stuff goes in `etc`” vibes.

Reply:
Oops! I intended to just drop the "lib". But "etc" is good.